src/Tizen.Applications.Common/Tizen.Applications/ApplicationInfo.cs

```diff
@@ -193,6 +193,28 @@ public string ApplicationType
             }
         }
 
+        /// <summary>
+        /// Gets the application component type.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public AppComponentType ApplicationComponentType
```

Comment:
```suggestion
        public AppComponentType ComponentType
```
because it already in ApplicationInfo class so `Application` is duplication of word